uses the xsi-compliant strerror in the else block below instead
--- a/src/misc.c
+++ b/src/misc.c
@@ -1956,7 +1956,7 @@
 #if defined(_WIN32)
     strerror_s(buf, buflen, err_num);
     return buf;
-#elif defined(__linux__) && defined(_GNU_SOURCE)
+#elif 0 && defined(__linux__) && defined(_GNU_SOURCE)
     /* GNU extension on Linux */
     return strerror_r(err_num, buf, buflen);
 #else
